Assets/RGLUnityPlugin/Scripts/PointCloudVisualization.cs

```diff
@@ -12,14 +12,52 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine.Serialization;
 using UnityEngine;
 
 namespace RGLUnityPlugin
 {
     [System.Serializable]
     public class PointCloudVisualization : MonoBehaviour
     {
-        public Material material;
+        public enum PointShape
+        {
+            FlatSquare = 0,
+            Box = 1,
+            Pyramid = 2
+        }
+
+        static private readonly List<Color> rainblowColors = new List<Color> {
+            Color.red,
+            new Color(1, 0.5f, 0, 1), // orange
+            Color.yellow,
+            Color.green,
+            Color.blue,
+            new Color(0.5f, 0, 1, 1) // violet
+        };
+
+        [SerializeField]
+        private PointShape pointShape = PointShape.Box;
+
+        [SerializeField]
+        [Range(0, 0.5f)]private float pointSize = 0.05f;
```

Comment:
In addition, it would be good to make a constraint to be a positive number. Otherwise, setting a zero has the same effect as disabling visualization, which is a doubled feature.
```suggestion
        [SerializeField]
        [Range(0, 0.5f)]
        private float pointSize = 0.05f;
```